src/uiHandler.luau

```diff
@@ -103,9 +107,25 @@ local function MainUI(props: {
 		}),
 
 		e(StudioComponents.Label, {
-			Text = `WakaTime API key{if util.isValidWakaTimeApiKey(keyText)
-				then ""
-				else " <b>(Please paste your API key that starts with waka_)</b>"}`,
+			Text = "WakaTime API URL",
+			TextXAlignment = Enum.TextXAlignment.Left,
+			Size = UDim2.new(1, 0, 0, 30),
+			RichText = true,
+		}),
+		e(StudioComponents.TextInput, {
+			Text = urlText,
+			OnChanged = function(newText)
+				setUrlText(newText)
+				props.setApiUrl(newText)
+			end,
+			ClearTextOnFocus = false,
+			PlaceholderText = "https://api.wakatime.com/api/v1"
+		}),
+
+		-- VerticalSpace({ Height = 10 }),
+
+		e(StudioComponents.Label, {
+			Text = "WakaTime API Key",
```

Comment:
I think there should still be some indication that users should add an API key when this is blank, that's why before this there was bold text "Please paste your API key" - we can have that while not mentioning waka_